app/src/organisms/ProtocolSetup/RobotCalibrationStep.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { Text, SPACING_3 } from '@opentrons/components'
+
+import type { Dispatch, State } from '../../redux/types'
+import type { ViewableRobot } from '../../redux/discovery/types'
+import * as PipetteOffset from '../../redux/calibration/pipette-offset'
+import * as Pipettes from '../../redux/pipettes'
+import * as PipetteConstants from '../../redux/pipettes/constants'
+import * as TipLength from '../../redux/calibration/tip-length'
+import { DeckCalibration } from './DeckCalibration'
+import { formatLastModified } from '../../organisms/CalibrationPanels/utils'
+
+interface Props {
+  robot: ViewableRobot
+}
+
+export const RobotCalibrationStep: React.FunctionComponent<Props> = (
```

Comment:
```suggestion
export function RobotCalibrationStep(props: Props): JSX.Element {
```
to follow the existing typing convention for functional components in the app